libraries/AP_Mount/STorM32_lib.h

```diff
@@ -0,0 +1,255 @@
+#pragma once
+
+#include <AP_AHRS/AP_AHRS.h>
+
+#define SERIAL_RECEIVE_BUFFER_SIZE      96 //the largest RCcmd response can be 77
```

Comment:
Make this more unique, something like `STORM32_LIB_SERIAL_RECEIVE_BUFFER_SIZE`

Reply:
ok